cloud/shared/bin/lib/check.py

```diff
@@ -47,9 +47,8 @@ def _semver_string_to_int_array(self, semver_string):
 
     def version_greater_than(self, min_version, test_version):
```

Comment:
Oh I ran into issues with this too. I have: https://github.com/seattle-uat/civiform/pull/2420

Reply:
Sounds good. Once that's merged, I'll revert these changes!

Reply:
I'll pre-emptively revert the version number changes since @jeanniefu is going to merge #2420.